### DISCUSSION TITLE
Account Pause Duration

### DISCUSSION THREAD

**Discussion by mbparvezme** [vercel/community#5701] [Account Issue] [answered]
### Question

Hi everyone,
My Vercel account was paused after exceeding the **function execution limit** on Jan 23, 2024. **Could someone please let me know when I can expect it to be un-paused?**

Thanks!

**Comment by swarnava** [comment, marked as answer]
Hi,

We have unblocked your account as onetime exception. Please read our Fair Usage Policy for Hobby plan: https://vercel.com/docs/concepts/limits/fair-use-policy

Thanks!

**Comment by mbparvezme** [comment]
Hello Swarnava,

Thank you for your response. Unfortunately, the issue hasn't been resolved
yet. The account still shows as "paused." Please let me know how we can
proceed to resolve this issue.

I want to mention that one of my sites was under a DDoS attack, which
caused the exception. Since such events are unpredictable and beyond our
control, how can we manage this situation within the confines of the free
subscription?

Thank you.

**Reply by swarnava** [reply]
Can you create a Billing ticket using https://vercel.com/help? Our team will triage and get back to you.

**Reply by mbparvezme** [reply]
Creating a Billing ticket is not allowed in the Hobby plan, I think!

**Reply by swarnava** [reply]
It should be allowed. 

<img width="925" alt="image" src="https://github.com/vercel/community/assets/1162991/1ffe0ea0-cd7d-4ee1-9974-d461b3e9e57d">

**Comment by prajjwal2-3** [comment]
Question

Hey everyone,

One of my website on vercel account was under ddos attack. This has happened twice with me and all of my deployments have been disabled.
![image](https://github.com/vercel/community/assets/121849327/b5190425-8c45-484a-9347-d9b358076e3c)
these are the stats, can someone please tell me when can i expect it to be un-paused.
@swarnava  can you please give it a look.

**Reply by swarnava** [reply]
Can you create a `Usage` ticket using https://vercel.com/help? Our team will triage and get back to you.
